### ISSUE TITLE
website relaunch improvements

### ISSUE THREAD

**Issue by felixhaeberle** [inlang/monorepo#1463]
> This is an issue which should collect improvements for the landing page:

`/`

- [ ] language switcher not working. can't click on it.
- [ ] font sizes & padding are too big. (vgl. old Footer) 
> cc @floriandwt we should probably make our base font size to 14px as we discussed on Friday
- [ ] cards in Hero Slider are not clickable -> I would imagine them to be clickable
- [ ] hero slider a bit too slow. would go for 3000ms instead of 4000ms
- [ ] section spacing & paddings overall on mobile is too big, there are huge gaps between sections & look at the gaps of the logo section in the header on mobile
- [ ] "Learn more about Lix" in Lix section could be a button. It's could practice to have a primary in every section. we can also decide to not make it a button, but I think as a link it won't be clicked that often

---

`/editor`

- [ ] Header on "/editor" not really clear to me. What are "Editor Projects" & "Apps". 
- [ ] "Use inlang CLI" on "/edtior" needs to be removed
- [ ] "/editor" -> "To access the editor, you must have the project.inlang.json file in your repository." -> change wording to "To access the editor, please create the project.inlang.json file in your repository."

--- 

`/documentation`

- [ ] Link is called "Developers" but URL is `/documentation`. Can we agree on one?
- [ ] I don't understand why "Document" is highlighted in the docs, or why this UI element is even there.
- [ ] GitHub & Discord link in Header on "Developers" section missing
- [ ] "Guides are in the Marketplace" -> confusing. Which Guides? What Marketplace? Looks like an Alert but not a clickable Card. Can we change the wording to make it more precise + add a text link ?

---

`/all`

- [ ] Link is called "All" but URL is `/marketplace` - can we agree on one?
- [ ] featured section on mobile not clear that it's "featured"
- [ ] "For translators" / "For Developers" -> styling could be better. simply a badge "Featured"?
- [ ] padding & font-sizes on mobile look way too big & off. can at least be shrinked to 50% most of the font sizes & gaps.

---

`/document` / `/email` / `/website` ...

- [ ] "No results yet" is really not good. I guess 95% of people will drop off at this point.
- [ ] No information on what the product is or wants to achieve or which problem it solves or is available or WHATEVER. really bad. 100% drop-off.
- [ ] "Items that might interest you" -> based on what? completely useless unless we don't know how the project is set up or we are in the context of another module. clear "future UI" case which does not apply currently. 
- [ ] why is `paraglide-js` or `vs-code-extension` not displayed in `/website`? 

--- 

`/application`

- `paraglide.js` coming soon highlighting section looks broken

**Comment by felixhaeberle** [comment]
furthermore, 

- [ ] we should have a look at the lighthouse score

<img width="561" alt="Bildschirmfoto 2023-10-14 um 13 04 46" src="https://github.com/inlang/monorepo/assets/34959078/af9dcb85-1e75-4f5d-a315-fbe66f8b4ee1">

- [ ] and fix the of images for sharing on social media

<img width="1276" alt="Bildschirmfoto 2023-10-14 um 13 09 00" src="https://github.com/inlang/monorepo/assets/34959078/5f070a1a-2db5-44e5-b214-5ba1e5c6ab3c">

**Comment by felixhaeberle** [comment]
@inlang/website if your agree, you can use:

<img width="420" alt="Bildschirmfoto 2023-10-14 um 13 18 34" src="https://github.com/inlang/monorepo/assets/34959078/f9c74c64-32e0-4aa2-8a4d-ce0041793b07">

**Comment by floriandwt** [comment]
@inlang/website I already started with adjusting the sizes: https://github.com/inlang/monorepo/pull/1460 
Let's test on other screens and implement the feedback to ensure quality and maintainability for the future.

### Regarding the font size 
**TLDR: I just feel like it's wrong to make the font smaller than needed just because everything **now** is too big.** 

As I already have shown, every website that has a sense for accessibility takes 16px.
I can send you thousands of arguments for it (https://www.learnui.design/blog/mobile-desktop-website-font-size-guidelines.html, using 16px -> https://developer.mozilla.org/en-US/docs/Web/API/setTimeout ...)

14px looks too small on my screen. Samuels screen seems to have other settings because not only our site but every site he visited looked way to big imo.

Let's look at the facts: https://www.who.int/news-room/fact-sheets/detail/blindness-and-visual-impairment 
> Globally, at least 2.2 billion people have a near or distance vision impairment.

So it doesn't matter that these people can't read anything when visiting on our site? We'll surely lose users when we don't find a good balance!

@samuelstroschein I will test it for MacBooks with smaller screen sizes and then ask you to test it again with 16px ok? If that doesn't work, I'm totally ok with changing it to 14px…

**Comment by samuelstroschein** [comment]
> TLDR: I just feel like it's wrong to make the font smaller than needed just because everything now is too big.

I think desktop should be 14px while mobile should be 16px.

The text we are writing here on GitHub is also 14px. But, let's go step by step. Getting the scale of the website to work is already a big step up.

**Comment by floriandwt** [comment]
> and fix the of images for sharing on social media

Just FYI: This is part of paraglide's implementation, we can't fix that if paraglide isn't working on the website yet.

**Comment by samuelstroschein** [comment]
Before continuing to work on this issue, https://github.com/inlang/internal/discussions/172 should be considered, which severely impacts this issue (potentially even redundant)

**Comment by felixhaeberle** [comment]
Most of the things do not apply anymore. I'll keep it open until the website restructuring is complete & will check again or close it then.

**Comment by NilsJacobsen** [comment]
@felixhaeberle we did watch through it will creating the new PR. Curious about review on the new one -> https://github.com/inlang/monorepo/pull/1471
